test/test_other.py

```diff
@@ -9730,6 +9730,16 @@ def test_noderawfs_access_abspath(self):
     self.run_process([EMCC, 'access.c', '-sNODERAWFS'])
     self.run_js('a.out.js', args=[os.path.abspath('foo')])
 
+  def test_noderawfs_readfile_prerun(self):
+    create_file('foo', 'bar')
+    create_file('main.c', r'''
+      int main() { return 0; }
+    ''')
```

Comment:
Maybe just use `test_file('hello_world.c'`)` instead of creating a new empty main.c here?

Reply:
Ah, good point! I initially thought the `expected_output` argument of `do_runf()` always did an identical assert, which could conflict with the `printf()` in that test file.  However, that only happens if you explicitly pass `assert_identical=True`.

Fixed with commit b5801e362949a5c5f5e16c7506942819c435d92e.